tests/integration_tests.rs

```diff
@@ -0,0 +1,384 @@
+        Ok(peridio_cloud_ca_dir) => PathBuf::from(peridio_cloud_ca_dir),
+        Err(var_error) => match var_error {
+            VarError::NotPresent => peridio_cloud_path().join("test/fixtures/ssl"),
+            VarError::NotUnicode(_) => {
+                panic!("peridio_cloud_CA_DIR environment variable is not valid unicode")
+            }
+        },
+    }
+}
+
+fn peridio_cloud_path() -> PathBuf {
+    let path = match env::var("PERIDIO_CLOUD_PATH") {
+        Ok(peridio_cloud_path) => PathBuf::from(peridio_cloud_path),
+        Err(var_error) => match var_error {
+            VarError::NotPresent => fs::canonicalize("../peridio-cloud").unwrap(),
+            VarError::NotUnicode(_) => {
+                panic!("PERIDIO_CLOUD_PATH environment variable is not valid unicode")
+            }
+        },
+    };
+
+    if path.is_dir() {
+        path
+    } else {
+        panic!(
+            "peridio_cloud path ({}) is not a directory",
+            path.to_str().unwrap()
+        );
+    }
+}
+
+fn random_name() -> String {
+    format!(
+        "peridio-cli-{}",
+        Uuid::now_v7()
+            .hyphenated()
+            .encode_upper(&mut Uuid::encode_buffer())
+    )
+}
+
+struct User {
+    username: String,
+    email: String,
+    prn_temp_path: TempPath,
+}
+impl User {
+    fn create(email_domain: &str) -> Self {
+        let username = random_name();
+        let email = format!("{}@{}", username, email_domain);
+        let password = "peridio-cli";
+        let prn_temp_path = NamedTempFile::new().unwrap().into_temp_path();
+
+        match peridio_cloud_core_mix_command()
+            .arg("peridio.user.create")
+            .args(&["--username", username.as_str()])
+            .args(&["--email", email.as_str()])
+            .args(&["--password", password])
+            .args(&["--prn-file", prn_temp_path.to_str().unwrap()])
+            .output()
+        {
+            Ok(output) => {
+                if !output.status.success() {
+                    panic!(
+                        "mix peridio.user.create failed\n  STDOUT:\n    {}\n\n  STDERR:\n    {}",
+                        indent_by(4, String::from_utf8(output.stdout).unwrap()),
+                        indent_by(4, String::from_utf8(output.stderr).unwrap())
+                    );
+                }
+            }
+            Err(error) => panic!("{:?}", error),
+        }
+
+        Self {
+            username,
+            email,
+            prn_temp_path,
+        }
+    }
+
+    fn create_api_key(&self) -> String {
+        let note = random_name();
+        let user_token_temp_path = NamedTempFile::new().unwrap().into_temp_path();
+
+        match peridio_cloud_core_mix_command()
+            .arg("peridio.user_token.create")
+            .args(&["--user-prn-file", self.prn_temp_path.to_str().unwrap()])
+            .args(&["--note", &note])
+            .args(&["--user-token-file", user_token_temp_path.to_str().unwrap()])
```

Comment:
Same comment as before ^^/

Reply:
They need to all be `&str`

![Screenshot 2024-02-27 at 10 24 33 AM](https://github.com/peridio/peridio-cli/assets/298259/6d684255-ec9f-40c0-8f2f-12f96ccda706)
![Screenshot 2024-02-27 at 10 24 39 AM](https://github.com/peridio/peridio-cli/assets/298259/516f4238-57be-4a53-996b-a853aff1fd5e)
![Screenshot 2024-02-27 at 10 24 51 AM](https://github.com/peridio/peridio-cli/assets/298259/02b4e601-5513-4fa7-94b3-011f777e3423)

Reply:
`&` removed from in front of `[`.